src/fides/data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -1,4 +1,401 @@
 privacy_notices:
+### GPP State-by-state ###
+- name: Data sales and sharing
+  notice_key: data_sales_sharing_gpp_us_state
+  id: pri_8a6a69d1-d0d1-4e5f-89c4-d17f448e0f6e_dss_gpp_state
+  description: We may transfer or share your personal information to third parties in exchange for monetary or other valuable consideration or for the purposes of [cross-context behavioral targeted advertising]. You can learn more about what information is used for this purpose in our privacy notice.
+  internal_description: Data sales and sharing (GPP, California) # TODO: update?
+  regions:
+    - us_ca
+  consent_mechanism: opt_out
+  data_uses:
+    - marketing.advertising.first_party.targeted
+    - marketing.advertising.third_party.targeted
+  enforcement_level: not_applicable
+  disabled: True
+  has_gpc_flag: False # TODO: not sure?
```

Comment:
Lots of things to check with product still looks like before release

Reply:
ok i synced up with Michael and i think we've got it all covered! he'll take another pass but i believe we should be good to go on this front now, unless anything is still sticking out to you.